archive/frames/views.py

```diff
@@ -139,8 +151,11 @@ def _get_aggregate_values(query_set, field, aggregate_field):
         else:
             return []
 
-    @list_route()
+    @action(detail=False)
     def aggregate(self, request):
+        """
```

Comment:
This endpoint takes a query parameter `aggregate_field`, which if set to a valid field name on which to aggregate, the response will only contain aggregations for that field. This query parameter is missing from the docs.

Reply:
missed that. it's documented now https://mgdaily.github.io/observatorycontrolsystem.github.io/assets/html/science-archive.html

Reply:
Nice!